### DISCUSSION TITLE
LEDのランド剥がれ

### DISCUSSION THREAD

**Discussion by KazukiNagao** [Yowkees/keyball#424] [Q&A]
LEDのランドが剥がれてしまいました。
正常に動作させるにはどうしたらいいでしょうか？？

問題箇所は3点です。
写真青枠の赤線部分が問題箇所です。

![IMG_2270](https://github.com/Yowkees/keyball/assets/5966602/5dfea888-ead8-4a08-a111-c56d1ec53823)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

下記画像の同色どうし（緑、ピンク、オレンジ）の端子をゴム被覆導線やエナメル線でショートしていただければ復活できると思います。
一部、パターン配線の剥がれも見られますが回路を見た感じ、上記作業だけで大丈夫そうです。

お手数ですがご対応お願いいたします。
![KB44_LED3fix](https://github.com/Yowkees/keyball/assets/84209772/4bf2cf8c-0c9a-4da5-a0fb-9907ad7ce876)

**Comment by curuknagao** [comment]
ありがとうございます！無事LED点灯しました！

**Comment by Yowkees** [comment]
無事に修正できたようでよかったです！ご対応ありがとうございました。
ぜひ末永くご愛用いただければ幸いです。
